src/Microsoft.ML.TimeSeries/DTRootCauseLocalization.cs

```diff
@@ -0,0 +1,308 @@
+            public Column[] Columns;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Weight for getting the score for the root cause item.", ShortName = "Beta", SortOrder = 2)]
+            public double Beta = DTRootCauseLocalizationEstimator.Defaults.Beta;
+
+        }
+
+        /// <summary>
+        /// The input and output column pairs passed to this <see cref="ITransformer"/>.
+        /// </summary>
+        internal IReadOnlyCollection<(string outputColumnName, string inputColumnName)> Columns => ColumnPairs.AsReadOnly();
+
+        private readonly double _beta;
+
+        /// <summary>
+        /// Localization root cause for multi-dimensional anomaly.
+        /// </summary>
+        /// <param name="env">The estimator's local <see cref="IHostEnvironment"/>.</param>
+        /// <param name="beta">Weight for generating score.</param>
+        /// <param name="columns">The name of the columns (first item of the tuple), and the name of the resulting output column (second item of the tuple).</param>
+
+        internal DTRootCauseLocalizationTransformer(IHostEnvironment env, double beta = DTRootCauseLocalizationEstimator.Defaults.Beta, params (string outputColumnName, string inputColumnName)[] columns)
+            : base(Contracts.CheckRef(env, nameof(env)).Register(RegistrationName), columns)
+        {
+            Host.CheckUserArg(beta >= 0 && beta <= 1, nameof(Options.Beta), "Must be in [0,1]");
+
+            _beta = beta;
+        }
+
+        // Factory method for SignatureDataTransform.
+        internal static IDataTransform Create(IHostEnvironment env, Options options, IDataView input)
+        {
+            Contracts.CheckValue(env, nameof(env));
+            env.CheckValue(options, nameof(options));
+            env.CheckValue(input, nameof(input));
+            env.CheckValue(options.Columns, nameof(options.Columns));
+
+            return new DTRootCauseLocalizationTransformer(env, options.Beta, options.Columns.Select(x => (x.Name, x.Source ?? x.Name)).ToArray())
+                .MakeDataTransform(input);
+        }
+
+        // Factory method for SignatureLoadModel.
+        private static DTRootCauseLocalizationTransformer Create(IHostEnvironment env, ModelLoadContext ctx)
+        {
+            Contracts.CheckValue(env, nameof(env));
+            var host = env.Register(RegistrationName);
+            host.CheckValue(ctx, nameof(ctx));
+            ctx.CheckAtModel(GetVersionInfo());
+            return new DTRootCauseLocalizationTransformer(host, ctx);
+        }
+
+        private DTRootCauseLocalizationTransformer(IHost host, ModelLoadContext ctx)
+            : base(host, ctx)
+        {
+            var columnsLength = ColumnPairs.Length;
+            // *** Binary format ***
+            // <base>
+            // double: beta
```

Comment:
>// double: beta [](start = 12, length = 15)

Can you please add this line to the Save method as well?

Reply:
> > // double: beta [](start = 12, length = 15)
> 
> Can you please add this line to the Save method as well?

Have added.